crates/libcontainer/src/rootfs/mount.rs

```diff
@@ -546,29 +546,27 @@ impl Mount {
             PathBuf::from(source)
         };
 
-        if let Err(err) =
-            self.syscall
+        let max_retries: u32 = 3;
+        let mut retries = 0;
+        loop {
```

Comment:
I don't prefer using the infinity loop. Please use `for` instead of that, at least.